docs/sources/alerting/unified-alerting/migration.md

```diff
@@ -1,48 +1,29 @@
 +++
-title = "Opt-in to Grafana 8 alerting"
+title = "Migration to Grafana 8 alerting"
 description = "Enable Grafana 8 Alerts"
 weight = 115
 +++
 
-# Opt-in to Grafana 8 alerting
```

Comment:
Can users opt in to use legacy alerting if they want to? Or they can no longer use legacy alerting?

Reply:
Answering my question. I spoke to @pkolyvas, and users with older OSS and Enterprise installations as well as users with Enterprise will still have legacy dashboard alerting OOTB. Let's keep this info and not remove it.

I am working on alerting docs in another PR and will make the necessary changes. Thanks!